app/controllers/users_controller.rb

```diff
@@ -0,0 +1,59 @@
+class UsersController < ApplicationController
+  before_action :set_user, only: [:show, :edit, :update, :destroy]
+
+  def index
+    @users = User.all
+  rescue ActiveRecord::RecordNotFound => e
```

Comment:
Instead or rescue in every action add rescue_from method in application controller.

Reply:
Done, Bhai.